can/interfaces/neousys_wdt.py

```diff
@@ -0,0 +1,269 @@
+""" Neousys WDT_DIO CAN driver """
```

Comment:
Let's rename the interface as neousys and put all the code into a package at `can/interfaces/neousys`.

Reply:
So I make subdir under interfaces or just rename file and class..

Reply:
Sorry for the delay replying to you. Yes please make a subdir under `can/interfaces` and feel free to create as many modules within that.